src/connection.rs

```diff
@@ -309,3 +314,19 @@ where
         }
     }
 }
+
+#[cfg(all(test, feature = "tokio_socket"))]
+mod tests {
+    use crate::new_connection;
+    use crate::sys::protocols::NETLINK_AUDIT;
+    use netlink_packet_audit::AuditMessage;
+    use tokio::time;
+
+    #[tokio::test]
+    async fn connection_is_closed() {
+        let (conn, _, _) = new_connection::<AuditMessage>(NETLINK_AUDIT).unwrap();
+        let join_handle = tokio::spawn(conn);
+        time::sleep(time::Duration::from_secs(2)).await;
```

Comment:
Is 200ms enough?

Reply:
On my system even 1 ms is enough (not sleeping at all not), but I don't want to trigger false negative tests depending on the build machine. But I guess 200 ms is fine. Will adapt it.